base/server/share/conf/schema.ldif

```diff
@@ -687,3 +687,92 @@ attributeTypes: ( authorityKeyHost-oid NAME 'authorityKeyHost' DESC 'Authority K
 -
 add: objectClasses
 objectClasses: ( authority-oid NAME 'authority' DESC 'Certificate Authority' SUP top STRUCTURAL MUST ( cn $ authorityID $ authorityKeyNickname $ authorityEnabled $ authorityDN ) MAY ( authoritySerial $ authorityParentID $ authorityParentDN $ authorityKeyHost $ description ) X-ORIGIN 'user defined' )
+
+dn: cn=schema
```

Comment:
Is it necessary to define the schema twice? Is there any way to avoid that?

Reply:
There is precedent for LDAP-based profile storage and LWCAs.  I can't remember why I did it this way (possibly because the standalone file makes it easier for `ipa-server-upgrade` or human administrators to update the schema).  If we want to address it, we should file a separate ticket and do it separately.

Reply:
OK